Add note on contacting us to site footer

anaulin opened the pull request:
For https://github.com/techworkersco/techworkersco.github.io/issues/60.

This PR adds a note to the site footer to encourage readers to contact us with contributions or stories, with our email address and a link to GitHub issues. This will appear everywhere on the site, not just below newsletter issues. I also took the liberty of adding some emoji flair.

It looks like so:
<img width="858" alt="Screen Shot 2020-05-13 at 3 08 45 PM" src="https://user-images.githubusercontent.com/6729309/81870915-ad30a480-952b-11ea-8d10-9bca1e472beb.png">

Thoughts?

stationaery commented:
I absolutely ❤️ the ✊💛!

The text could be a bit more concise- something like, "Want to contribute
to this newsletter or share a story? We'd love to hear from you! Email
twcnewsletter@protonmail.com or open a new issue."

Also, noting for this and future tweaks: I think we could develop and apply
a stylesheet for everything- underlining links, mouseover color changes,
etc. are a bit inconsistent.

On Wed, May 13, 2020 at 3:10 PM Ana Ulin <notifications@github.com> wrote:

> @anaulin <https://github.com/anaulin> requested your review on: #66
> <https://github.com/techworkersco/techworkersco.github.io/pull/66> Add
> note on contacting us to site footer.
>
> —
> You are receiving this because your review was requested.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/techworkersco/techworkersco.github.io/pull/66#event-3334096049>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACES7PK5KGSV7ZHWU2OP5WTRRMLGVANCNFSM4NAE7XRA>
> .
>

anaulin commented:
@stationaery I updated the text as per your suggestion. Also removed the additional underlines on the existing footer links (hadn't noticed those before; if we want to have underlines in the links throughout the site, we can add those to the stylesheet, which we already have).

This is how it looks now:
<img width="1011" alt="Screen Shot 2020-05-13 at 3 58 58 PM" src="https://user-images.githubusercontent.com/6729309/81874473-cbe66980-9532-11ea-8491-16ffdd8ced2a.png">

dellsystem commented on _includes/footer.html:
```diff
@@ -3,8 +3,20 @@
-        This site is <u><a href="{{ site.links.github }}">open source</a></u>.
+        Want to contribute to this newsletter or share a story? We'd love to hear from you! Email
+        <a href="mailto:twcnewsletter@protonmail.com">twcnewsletter@protonmail.com</a>
+        or <a href="https://github.com/techworkersco/techworkersco.github.io/issues">open a new issue</a>.
```
could this be changed to "open a GitHub issue" or something to clarify what "issue" means here? (since it could also mean newsletter issues)

stationaery commented:
Agreed re. disambiguating 'issues' :p

Also, speaking of stylesheet, when the window is >825 and <1075, the body
text frame is pinched to a much more narrow width than even <800 and
>635. Not sure where layout is determined (didn't see it in the CSS file),
but happy to open an issue :)

On Wed, May 13, 2020 at 4:09 PM Wendy Liu <notifications@github.com> wrote:

> *@dellsystem* commented on this pull request.
> ------------------------------
>
> In _includes/footer.html
> <https://github.com/techworkersco/techworkersco.github.io/pull/66#discussion_r424781403>
> :
>
> > @@ -3,8 +3,20 @@
>  <hr width="50%">
>  <p class="text-muted text-center">
>      <small>
> -        A <u><a href="https://techworkerscoalition.org">Tech Workers Coalition</a></u> project.
> -        This site is <u><a href="{{ site.links.github }}">open source</a></u>.
> +        Want to contribute to this newsletter or share a story? We'd love to hear from you! Email
> +        <a href="mailto:twcnewsletter@protonmail.com">twcnewsletter@protonmail.com</a>
> +        or <a href="https://github.com/techworkersco/techworkersco.github.io/issues">open a new issue</a>.
>
> could this be changed to "open a GitHub issue" or something to clarify
> what "issue" means here? (since it could also mean newsletter issues)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/techworkersco/techworkersco.github.io/pull/66#pullrequestreview-411359977>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACES7PKERXW6ROACC2GGKADRRMSA3ANCNFSM4NAE7XRA>
> .
>

dellsystem commented:
I added a comment about the wording, but other than that, LGTM!

re @stationaery on stylesheet: we're just using bootstrap styles I believe. Color inconsistencies are probably the result of :visited vs non-visited links. I agree that it looks weird - I'd +1 a PR to settle on one color.

stationaery commented:
LGTM2!

@dellsystem, what say you re. window width <=> body text spacing? Is that
adjustable?

On Wed, May 13, 2020 at 4:12 PM Wendy Liu <notifications@github.com> wrote:

> I added a comment about the wording, but other than that, LGTM!
>
> re @stationaery <https://github.com/stationaery> on stylesheet: we're
> just using bootstrap styles I believe. Color inconsistencies are probably
> the result of :visited vs non-visited links. I agree that it looks weird -
> I'd +1 a PR to settle on one color.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/techworkersco/techworkersco.github.io/pull/66#issuecomment-628293067>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACES7PNCQ4WVSOKBBZN7PULRRMSPNANCNFSM4NAE7XRA>
> .
>

dellsystem commented:
I see your point @stationaery. That looks like a bootstrap thing. Want to open a separate issue for it?

jessesquires commented:
Links are consistent colors. It's just the difference between visited vs not visited 😄

jessesquires commented:
I think the footer text can be much more concise:


Want to contribute? [Email us](#) or [open a GitHub issue](#).

jessesquires commented:
Also worth noting: this footer text will not be included in the emails. 

I think that's fine, just wanted to point that out since it isn't obvious.

jessesquires commented on _includes/footer.html:
```diff
@@ -3,8 +3,20 @@
+
+<p class="text-muted text-center">
+    <small>
+        A <a href="https://techworkerscoalition.org">Tech Workers Coalition</a> project.
```
what about including all of the new text here, and using `<br>` ?

i think it might look nicer without all the extra spacing.

stationaery commented:
But we can certainly add the footer stuff to the MailChimp template!

On Wed, May 13, 2020 at 5:57 PM Jesse Squires <notifications@github.com>
wrote:

> *@jessesquires* commented on this pull request.
> ------------------------------
>
> In _includes/footer.html
> <https://github.com/techworkersco/techworkersco.github.io/pull/66#discussion_r424812488>
> :
>
> > @@ -3,8 +3,20 @@
>
>  <hr width="50%">
>
>  <p class="text-muted text-center">
>
>      <small>
>
> -        A <u><a href="https://techworkerscoalition.org">Tech Workers Coalition</a></u> project.
>
> -        This site is <u><a href="{{ site.links.github }}">open source</a></u>.
>
> +        Want to contribute to this newsletter or share a story? We'd love to hear from you! Email
>
> +        <a href="mailto:twcnewsletter@protonmail.com">twcnewsletter@protonmail.com</a>
>
> +        or <a href="https://github.com/techworkersco/techworkersco.github.io/issues">open a new issue</a>.
>
> +    </small>
>
> +</p>
>
> +
>
> +<p class="text-center">
>
> +    ✊🏼💛
>
> +</p>
>
> +
>
> +<p class="text-muted text-center">
>
> +    <small>
>
> +        A <a href="https://techworkerscoalition.org">Tech Workers Coalition</a> project.
>
>
> what about including all of the new text here, and using <br> ?
>
> i think it might look nicer without all the extra spacing.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/techworkersco/techworkersco.github.io/pull/66#pullrequestreview-411395924>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACES7POGWBDSPQHSTNU7SNLRRM6WVANCNFSM4NAE7XRA>
> .
>

anaulin commented:
I addressed everyone's comments. This is what it looks like now:

<img width="689" alt="Screen Shot 2020-05-13 at 6 57 33 PM" src="https://user-images.githubusercontent.com/6729309/81884098-cb0e0180-954b-11ea-8e43-c5572a3c701f.png">

stationaery commented on _includes/footer.html:
```diff
@@ -3,18 +3,12 @@
-
-<p class="text-muted text-center">
-    <small>
+        Want to contribute? <a href="mailto:twcnewsletter@protonmail.com">Email</a> us or
```
put the 'us' inside the <a href> tag

jessesquires commented:
Looks great!

+1 for linking “us”

***

One last thing: should we include all of the emoji skin tones for the fist? 😅😅😅 I don’t know what “best practices” are with that...

✊✊🏻✊🏼✊🏽✊🏾✊🏿

anaulin commented:
I put the "us" inside the link, and changed the fist to be of the "default" skin tone (aka weird yellow). I hadn't realized I had inserted an emoji with a skin tone choice, that was unintentional.

I don't know what "best practices" are about the skin tone, but I figure the "default" is the most neutral. I am a little weary of including the ones with the different skin tone modifiers, because I don't know how well will they be displayed on everyone's systems (since I'm using actual unicode emoji, and not inserting images, on some systems people might see the weird "skin patch + fist" display you get on some older systems or apps). But maybe I'm worrying too much about that? Happy to add more firsts of different kinds, if folks like that idea.

This is what it looks like now:
<img width="682" alt="Screen Shot 2020-05-13 at 10 18 09 PM" src="https://user-images.githubusercontent.com/6729309/81895962-585f4f00-9568-11ea-89b5-bc62bf03351c.png">

jessesquires commented:
@anaulin perfect! 💯
